META

```diff
@@ -6,5 +6,5 @@ Release:       1
 Release-Tags:  relext
 License:       CDDL
 Author:        OpenZFS on Linux
-Linux-Maximum: 5.4
+Linux-Maximum: 5.6
```

Comment:
should this be 5.7?

Reply:
no

Reply:
I chose not bump it to 5.7 because this kernel release has not yet been finalized and is still subject to change.